test-fixture/src/lib.rs

```diff
@@ -133,6 +133,25 @@ pub fn default_server() -> Connection {
     make_default_server(DEFAULT_ALPN)
 }
 
+/// Create a transport server with an initial `max_stream_data` value.
+#[must_use]
+pub fn default_server_max_stream_data(max_stream_data: u64) -> Connection {
```

Comment:
I think that it might be better to have this accept ConnectionParameters instead.  That seems more broadly useful.